pull-submodules.py

```diff
@@ -177,10 +177,10 @@ def main():
                                     branchToPull = "master"
                                     if branchExistsPcs.stdout.decode("utf-8").rstrip("\n") != "0":
                                         branchToPull = frameworkBranchName
-                                        print(" --> Pulling branch : " + branchToPull + "  ", end='')
+                                    print(" --> Pulling branch : " + branchToPull + "  ", end='')
```

Comment:
Just a reminder that we should use Python f-strings whenever possible 👍🏻 

```suggestion
                                    print(f" --> Pulling branch: {branchToPull}", end='')
```

Reply:
> Just a reminder that we should use Python f-strings whenever possible 👍🏻

@DavidDiezIb did you click on "commit suggestion" or did you write the commit yourself? just wondering in case you are not aware that you can accept suggestions from the web gui

Reply:
I did it myself, good to know for the next time ;)